ansible/roles/service/hippo_authoring/files/hippo_authoring.yml

```diff
@@ -0,0 +1,280 @@
+---
+
+Description: Hippo Authoring stack
+AWSTemplateFormatVersion: 2010-09-09
+
+Parameters:
+  HostedDomainPublic:
+    Description: Environment public domain name
+    Type: String
+
+  Image:
+    Description: AMI ID of the
+    Type: AWS::EC2::Image::Id
+
+  InstanceCount:
+    Description: Number of EC2 instances to launch
+    Type: Number
+    Default:  1
+
+  InstanceMonitoring:
+    Description: Should the detailed instance monitoring be enabled
+    Type: String
+    Default: false
```

Comment:
Why disabled by default?

Reply:
$$ - we don't need it by default on all environments